src/components/CardBtnDefiner/CardBtnDefiner.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import PropTypes from 'prop-types';
+
+import Toggler from "../Toggler/Toggler"
+import CardStatus from "../CardStatus/CardStatus";
+import ChooseDayBtn from "../ChooseDayBtn/ChooseDayBtn";
+
+import { CardStatuses } from "../../config/enums";
+
+const CardDefiner = ({status, handleToggler, addDayToTask, checkedToggler, isDone}) => {
+
+    if (status === CardStatuses.ChooseDayBtn) {
+        return <span onClick={addDayToTask}>
+                <ChooseDayBtn />
+            </span>
+    }
+    else if (status === CardStatuses.Toggle) {
+        return <span onClick={handleToggler}>
+                <Toggler checkedToggler={checkedToggler} />
```

Comment:
I think `handleToggler` shoud be passed into Toggler component and span is redundant here, same for `ChooseDayBtn`